modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -439,6 +451,59 @@ const FDSEntries = ({
 		navigate(url);
 	};
 
+	const onDeleteClick = ({
+		itemData,
+		loadData,
+	}: {
+		itemData: FDSEntry;
+		loadData: Function;
+	}) => {
+		openModal({
+			bodyHTML: Liferay.Language.get(
+				'deleting-a-dataset-is-an-action-that-cannot-be-reversed'
+			),
+			buttons: [
+				{
+					autoFocus: true,
+					displayType: 'secondary',
+					label: Liferay.Language.get('cancel'),
+					type: 'cancel',
+				},
+				{
+					displayType: 'danger',
+					label: Liferay.Language.get('delete'),
+					onClick: ({processClose}: {processClose: Function}) => {
```

Comment:
I don't know what was going through my mind when I exposed close modal message as `processClose` :disappointed:  It's such an awkward name. It makes sense internally, because we are processing closing, but not as external API. We should add `closeModal` at some point, as an alias. Not a blocker for this PR.